MailManager/src/main/java/inc/softserve/EmailServiceApplication.java

```diff
@@ -37,5 +28,6 @@ public void run(EmailServiceConfiguration configuration,
         Injector injector  = Guice.createInjector(new EmailServiceModule(configuration));
         environment.jersey().register(injector.getInstance(EnvelopeResource.class));
         environment.healthChecks().register("health", injector.getInstance(MailManagerHealthCheck.class));
+        environment.healthChecks().register("mongoHealth", injector.getInstance(MongoHealthCheck.class));

```

Comment:
While that may have sense to provide more detailed health checks for each of the subsystem, I believe that main `/health` endpoint should do all the checks at once.